static/js/index.js

```diff
@@ -3592,6 +3592,19 @@ function getWeatherMarker (weather, ts) {
 }
 
 function getNestMarker (nest, geojson, ts) {
+    const pkmn = masterfile.pokemon[nest.pokemon_id];
+    let typesIcon = '';
+    if (pkmn) {
+        const types = pkmn.types;
+        if (types && types.length > 0) {
+            //if (types.length === 2) {
+                typesIcon += '<span class="text-nowrap"><img src="/img/nest/nest-' + types[0].toLowerCase() + '.png" height="56" width="auto" style="-webkit-mask-image: -webkit-gradient(linear, left 90%, left bottom, from(rgba(0,0,0,1)), to(rgba(0,0,0,0)))"></span>&nbsp;';
+            //    typesIcon += '<img src="/img/nest/nest-' + types[1].toLowerCase() + '.png" height="56" width="auto" style="-webkit-mask-image: -webkit-gradient(linear, left 90%, left bottom, from(rgba(0,0,0,1)), to(rgba(0,0,0,0)))"></span>';
+            //} else {
+            //    typesIcon += '<img src="/img/nest/nest-' + types[0].toLowerCase() + '.png" height="56" width="auto" style="-webkit-mask-image: -webkit-gradient(linear, left 90%, left bottom, from(rgba(0,0,0,1)), to(rgba(0,0,0,0)))">';
+            //}
+        }
+    }
```

Comment:
1. Probably best to move CSS to `index.css` or something.
2. Do not use `width`/`height` in HTML, prefer CSS. (it might be fine for `img` but some other tags might not support this)
3. Multi-type, either remove it or mark PR as draft.